SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -80,7 +84,11 @@ private static void AddSetting(AnalysisProperties properties, string id, string
             {
                 properties.Add(new Property() { Id = id, Value = value });
             }
+        }
 
+        private static bool IsSecuredServerProperty(string s)
+        {
+            return s.EndsWith(".secured", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) 'AnalysisConfigGenerator.IsSecuredServerProperty(string)' passes 'StringComparison.InvariantCultureIgnoreCase' as the 'StringComparison' parameter to 'string.EndsWith(string, StringComparison)'. To perform a non-linguistic comparison, specify 'StringComparison.Ordinal' or 'StringComparison.OrdinalIgnoreCase' instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AUseOrdinalStringComparison)